letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -110,7 +125,7 @@ su_sudo() {
 
 SUDO_ENV=""
 export CERTBOT_AUTO="$0"
-if [ -n "${LE_AUTO_SUDO:+x}" ]; then
```

Comment:
Not sure why this was in there, but it appears to expand `LE_AUTO_SUDO` to `x`, unless `LE_AUTO_SUDO` is empty. But `-n` checks if the string is empty, so it gives the same result regardless.
Maybe to guard against some ancient shell?
But an ancient shell would likely not support the `${y:+x}` syntax ...